### ISSUE TITLE
Version for SuiteScript 2.0?

### ISSUE THREAD

**Issue by drernie** [suiteplus/nsmockup#16]
Very cool!  Do you have any plans to upgrade this for SS2.0?  Or are you aware of any other projects that attempt to do that?

**Comment by NicolasRitouet** [comment]
@drernie have you been able to find something suitable for SuiteScript 2.0?
I'm also looking for this.

**Comment by drernie** [comment]
This is the best I found.
https://github.com/headintheclouddev/typings-suitescript-2.0

**Comment by cancerberoSgx** [comment]
@drernie - that's a nice project but doesn't mock - just has ss2 typescript interfaces. I always wonder if it is a good idea to mock netsuite locally or if the code should run in the platform itself-  what do you think?

**Comment by drernie** [comment]
I agree using the Sandbox is best. It still would be nice to be able to run
some sort of continuous integration on our scripts offline.

**Comment by cancerberoSgx** [comment]
that would be ideal and if you can isolate some logic from suitescript you can. In the real world I'm only able to do so in no more than 30% of my code because i have suitescripts everywhere. for the rest of the code you need to mock suitescript some how. if you have any idea please shere :)

**Comment by NicolasRitouet** [comment]
Anyone is using [this library](
https://github.com/3EN-Cloud/netsumo)?

We're considering open-sourcing our collection of SS2.0 mocks. It would be great to join our forces 👍
